common.js

```diff
@@ -535,14 +535,15 @@ const getErrorHandler = (fromCli = false) => {
 
 /**
  * Reads the contents of a zip file and return an object with key being the filename and value being the contents
- * @param {string} path zip file path
+ * @param {string|Buffer} pathOrBuffer zip file path or buffer
  * @returns {Promise<Record<string, string>>}
  */
-const readZipFileContents = path => {
+const readZipFileContents = pathOrBuffer => {
+  const readZip = pathOrBuffer instanceof Buffer ? yauzl.fromBuffer : yauzl.open;
```

Comment:
Some checking should be done that this is either a buffer or path before trying to open it. If not, then an error should be thrown.